src/gateway_gatt_char_wifi_ssid.erl

```diff
@@ -33,7 +33,8 @@ init(Path, _) ->
          {gatt_descriptor_pf, 1, [utf8_string]}
         ],
     {ok, SignalID} = connman:register_state_notify({tech, wifi}, self(), Path),
-    {ok, Descriptors, update_value(#state{path=Path, wifi_signal=SignalID})}.
+    NextState = update_value(#state{path=Path, wifi_signal=SignalID}, "disconnect"),
+    {ok, Descriptors, update_value(NextState, "online")}.
```

Comment:
What does setting the value from disconnect to online in two consecutive lines achieve?

Reply:
It forces the process to go through both the `on_online_state` and `on_offline_state` code paths in succession rather than just one which is more or less how the original code worked before I changed it. We haven't received an event so we don't know what Wi-Fi state we're transitioning to. Need to check for all of them.

Reply:
Maybe leave a comment to that effect?

Reply:
Done.